src/server/pkg/metrics/segment.go

```diff
@@ -23,6 +24,28 @@ func reportSegment(metrics *Metrics) {
 		},
 	})
 	if err != nil {
-		lion.Errorf("error reporting to Segment: %s", err.Error())
+		lion.Errorf("error reporting cluster metrics to Segment: %s", err.Error())
+	}
+}
+
+func identifyNewUser(userID string) {
```

Comment:
I'm a bit confused, is there an `identifyExistingUser` or something that I'm not seeing somewhere? This function isn't called anywhere so it's a little tricky to understand how it's going to be used. How does one figure out if they have a new user or one that segment already knows about?